tutor/src/screens/my-courses/dashboard/index.tsx

```diff
@@ -114,21 +114,11 @@ export const MyCoursesDashboard = () => {
     const courses = useAllCourses()
 
     const [deleteOfferingIdModal, setDeleteOfferingIdModal] = useState<ID | null>(null)
-    const [displayedOfferingIds, setDisplayedOfferingIds] = useState<ID[]>(UiSettings.get('displayedOfferingIds') || [])
+    const [displayedOfferingIds, setDisplayedOfferingIds] = useState<ID[]>(
+        useDisplayedCourseOfferingIds()
+    )
     const [isEditMode, setIsEditMode] = useState<boolean>(false)
 
-    // First time setting the `displayedOfferingIds`
-    useEffect(() => {
-        if (offerings.length > 0 && courses.length > 0) {
-            const uiDisplayedOfferingIds = UiSettings.get('displayedOfferingIds')
-            if (!uiDisplayedOfferingIds) {
-                const offeringIds = map(filter(offerings, o => some(courses, c => c.offering_id === o.id)), m => m.id)
-                UiSettings.set('displayedOfferingIds', offeringIds)
-                setDisplayedOfferingIds(offeringIds)
-            }
-        }
-    }, [offerings, courses])
-
```

Comment:
this had a bug where if UiSettings was empty, no offerings would be displayed.  The corrected logic always displays the offerings that are in use by courses + any of the ids present in UiSettings